dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/src/Microsoft.ApplicationModel.Resources.vcxproj

```diff
@@ -124,22 +104,32 @@
       <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
       <ModuleDefinitionFile>Microsoft.ApplicationModel.Resources.def</ModuleDefinitionFile>
       <AdditionalDependencies>$(OutDir)..\mrm\MRM.lib;onecoreuap.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <!-- This sets /DEBUG -->
+      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <!-- Note: /PROFILE implies the following 4 settings.
+           However, since /PROFILE is only supported on VS Enterprise we will also set them explicitly below as well. -->
```

Comment:
>we will also set them explicitly below as well [](http://example.com/codeflow?start=70&length=46)

why? does /Profile not work in the APIScan task in the build pipeline?

Reply:
The build pipeline uses VS Enterprise, so it's not an issue.
I wanted to set them here in case an external person is building with VS Community edition.
I think we'd still want to set `EnableCOMDATFolding=false` so they have a good debugging experience and also so that we don't end up with differences in the binary based on what edition of VS you use to build.

Reply:
don't think it's a big deal. if you want that, just don't set profile.

Reply:
Hmm, I'm not sure I understand... maybe I'm missing something.
The vcxproj file by default sets `/profile`, so someone would need to edit/change to remove it.
By settings these as well that way VS Community builds would be closer to the same as VS Enterprise builds, which I think would be a good thing... (?).

Reply:
with the other flags set explicitly, there is not need to set <profile>true</profile>

Reply:
I thought about doing that, but `/profile` also ends up setting `/debugtype:cv,fixup` and there is no explicit option to set that.
I think we could try adding additional command line options to get the same effect.
(Though the APIScan documentation seems to imply that `/profile` is the preferred way to do it.)

Reply:
I pushed changes which now set explicit flags, so we aren't using `/profile` anymore -- so there shouldn't be anymore issues with different versions of VS.